grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -127,4 +127,36 @@ class NewConsentGroupController extends AuthenticatedController {
         response.status = 200
         render([consent: consent] as JSON)
     }
+
+    Collection<ConsentCollectionLink> getConsentCollectionLinks() {
+        Issue issue = queryService.findByKey(params.consentKey)
+        Collection<ConsentCollectionLink> collectionLinks = queryService.findCollectionLinksByConsentKey(issue.projectKey)
+        render([collectionLinks: collectionLinks.linkedProject] as JSON)
```

Comment:
Should we need to handle a 404 here? It would be nice for completeness.

Reply:
I'm not sure if we want to handle a 404, this method is expected to return something or an empty collection. 
Since it's only used for listing Links this would need us modify the Ui to ignore this 404. If this doesn't rise any concerns to you,  i'll gladly add it.

Reply:
Makes sense then 👍